### ISSUE TITLE
Can TabNet be used in recommender systems?

### ISSUE THREAD

**Issue by EvanZ** [dreamquark-ai/tabnet#295]
Let's say we have a canonical movie rating dataset (really tuples of users and movies they watched). Currently one could use word2vec to do unsupervised learning on the data and learn embeddings for users and movies. I was thinking about using attention for this task, and just learned about TabNet. Is this something that TabNet can readily do using the pretrainer?

**Comment by Optimox** [comment]
Hello @EvanZ,
TabNet is not specifically suited for recommender systems. But there's one advantage over XGBoost for example, the TabNetMultiTaskClassifier can help you train a model that can recommend several items to a same user.

However, you'll need to treat the recommendation problem as a supervised problem.
